hbase-server/src/main/resources/hbase-webapps/regionserver/region.jsp

```diff
@@ -79,17 +86,20 @@
            <th>Len Of Biggest Cell</th>
            <th>Key Of Biggest Cell</th>
          </tr>
-       <%   for(StoreFile sf : storeFiles) { %>
+       <% int count = 0;
+          for(StoreFile sf : storeFiles) {
+            if (!fs.exists(sf.getPath())) continue;
```

Comment:
Do we need to check the file existence for primary region also?

Reply:
Actually, we don't need to perform this check for primary region beacause it is authoritative.

But compared to the following code.
```
if (region.getRegionInfo().getReplicaId() > RegionInfo.DEFAULT_REPLICA_ID && !fs.exists(sf.getPath()))
```
I think the former may be more concise.

Reply:
IMO it's worth to add a primary region check, we will save unnecessary RPCs to Namenode.

Reply:
Make sense, I will update it, thanks.